src/main/java/games/strategy/util/Version.java

```diff
@@ -64,6 +66,20 @@ public Version(final String version) {
     }
   }
 
+  /**
+   * Returns the exact and full version number.
+   * For example, if we specify:
+   * <code>
+   * new Version(1.2.3.4.5).getMicro == 4; // true
+   * new Version(1.2.3.4.5).toString().equals("1.2.3.4"); // true
+   * new Version(1.2.3.4.5).getExactVersion.equals("1.2.3.4.5"); // true
+   * </code>
+   */
+  public String getExactVersion() {
+    // in case of deserialization, exactVersion may be null, in which case toString() it.
+    return exactVersion != null ? exactVersion : toString();
```

Comment:
Yup, NP.  I just have "serialization concerns" on my mind while working the serialization proxy stuff. :smile: